pkg/orchestrator/pod_test.go

```diff
@@ -307,6 +312,7 @@ func TestExtractPodMessage(t *testing.T) {
 						Name: "bazName",
 					},
 				},
+				QOSClass: "BestEffort",
```

Comment:
out of curiosity, is it possible that a QOSClass is an empty string?

Reply:
The code would accept it, but the QoSClass is a field from the kubernetes api that is supposedly not-empty: https://godoc.org/k8s.io/api/core/v1#PodQOSClass